.github/workflows/build-backend.yml

```diff
@@ -0,0 +1,46 @@
+name: Build and Push Backstage Docker Image
+
+on:
+  repository_dispatch:
+    types: [backstage_release]
```

Comment:
Please specify the required `inputs` and their description (that is `version`, as it seems)